src/handlers/testing/exec/components/progress/index.tsx

```diff
@@ -5,6 +5,7 @@ import { EventName, emitter, type EventSchemas } from '../../util/emitter';
 import { AUTOBLOCKS_WEBAPP_BASE_URL } from '../../../../../util/constants';
 import { makeTestRunStatusFromEvaluations } from '../../util/evals';
 import { EvaluationPassed, TestRunStatus } from '../../util/models';
+import type { Handler } from '../../../../../util/types';
```

Comment:
Should we set up some path aliases so we don't have big relative imports like this?

Reply:
yeah i can do as a follow up